itou/templates/apply/includes/list_tr.html

```diff
@@ -78,18 +77,55 @@
     </td>
     {% if job_applications_list_kind is JobApplicationsListKind.SENT or job_applications_list_kind is JobApplicationsListKind.RECEIVED and request.current_organization.is_subject_to_eligibility_rules %}
         {# prescription or received by IAE employer #}
-        <td>
-            {% if job_application.preloaded_administrative_criteria %}
-                <ul class="mb-0">
-                    {% for criteria in job_application.preloaded_administrative_criteria %}<li>{{ criteria.name }}</li>{% endfor %}
-                </ul>
-            {% else %}
-                <i class="text-disabled">Aucun critère sélectionné</i>
-            {% endif %}
-            {% if job_application.preloaded_administrative_criteria_extra_nb %}
-                <span>+ {{ job_application.preloaded_administrative_criteria_extra_nb }} autres critères</span>
-            {% endif %}
-        </td>
+        {% if job_applications_list_kind is JobApplicationsListKind.SENT and not job_application.to_company.is_subject_to_eligibility_rules %}
+            <td>
+                <i class="text-disabled">Candidature hors-IAE</i>
+            </td>
+            <td>
+                <i class="text-disabled">Candidature hors-IAE</i>
```

Comment:
```suggestion
                <i class="text-disabled">Candidature hors IAE</i>
            </td>
            <td>
                <i class="text-disabled">Candidature hors IAE</i>
```

Clairement pas important, mais je vois pas pourquoi mettre un tiret ici 🤔